apache/superset
# chore: Remove TimeTable.less

### SUMMARY
Removes the TimeTable.less file from the TimeTable component

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
No visual changes

### TESTING INSTRUCTIONS
1. Open a Time-series Table chart
2. Make sure no visual changes can be observed

### ADDITIONAL INFORMATION
